src/services/services.tsx

```diff
@@ -0,0 +1,12 @@
+const backend_URL: string = 'https://lessoner.herokuapp.com/en';
```

Comment:
let's move that variable to another file and rename it

New name: `BACKEND_URL` (uppercase)

filepath: `src/constants.ts`

Reply:
done